### ISSUE TITLE
Logo SVG missing title

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#80]
The MDN logo SVG does not have a title that can be read out by screen readers.

Either add a title tag to the SVG along with `aria-labelled-by` or add an `aria-label` attribute. Both should be wrapped with `gettext` in order to be localizable.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563461
